# HG changeset patch
# User Kirill Smelkov <kirr@landau.phys.spbu.ru>
# Date 1216803753 -14400
# Node ID d4f20d32b03c441d0902c540796840c08b2861eb
# Parent  e73528b0abee9d8d4b82aebdb435520a410dff18
'make all' to build Scanners.so

And also clean compiled module(s) on 'make clean'

diff --git a/Makefile b/Makefile
--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,16 @@ PYTHON?=python
 PYTHON?=python
+
+all:	local
+
+local:
+	${PYTHON} setup.py build_ext --inplace
 
 clean:
 	@echo Cleaning Source
 	@rm -f *.pyc */*.pyc */*/*.pyc 
 	@rm -f *~ */*~ */*/*~
 	@rm -f core */core
+	@rm -f Cython/Plex/Scanners.{so,pyd}
 	@(cd Demos; $(MAKE) clean)
 
 testclean:
